Modelica/Electrical/Analog/Examples/Lines/CompareCmatrix.mo

```diff
@@ -0,0 +1,273 @@
+  connect(v2.p, v3.p)
+    annotation (Line(points={{28,6},{20,6},{20,-12},{28,-12}},
+                                                             color={0,0,255}));
+  connect(MTL_C.pn, ground.p) annotation (Line(points={{91.9,2.7},{86,2.7},{
+          86,-32}}, color={0,0,255}));
+  connect(c30.n, ground1.p)
+    annotation (Line(points={{-36,-32},{-36,-36},{-18,-36}}, color={0,0,255}));
+  connect(c20.n, ground1.p)
+    annotation (Line(points={{-54,-32},{-54,-36},{-18,-36}}, color={0,0,255}));
+  connect(c10.n, ground1.p)
+    annotation (Line(points={{-66,22},{-66,-36},{-18,-36}}, color={0,0,255}));
+  connect(c10.p, c12.p)
+    annotation (Line(points={{-66,42},{-66,52},{-18,52}}, color={0,0,255}));
+  connect(c12.n, c23.p)
+    annotation (Line(points={{-18,32},{-18,18}}, color={0,0,255}));
+  connect(v1a.p, v2a.p) annotation (Line(points={{-124,48},{-132,48},{-132,18},{
+          -124,18}}, color={0,0,255}));
+  connect(v2a.p, v3a.p) annotation (Line(points={{-124,18},{-132,18},{-132,-2},
+          {-124,-2}},color={0,0,255}));
+  connect(c12.p, c13.p)
+    annotation (Line(points={{-18,52},{0,52},{0,32}},     color={0,0,255}));
+  connect(c13.n, c23.n)
+    annotation (Line(points={{0,12},{0,-2},{-18,-2}},     color={0,0,255}));
+  connect(v1a.n, r1n.p)
+    annotation (Line(points={{-104,48},{-96,48}}, color={0,0,255}));
+  connect(r1n.n, c12.p) annotation (Line(points={{-80,48},{-66,48},{-66,52},{
+          -18,52}}, color={0,0,255}));
+  connect(v3a.n, r3n.p)
+    annotation (Line(points={{-104,-2},{-96,-2}}, color={0,0,255}));
+  connect(v2a.n, r2n.p)
+    annotation (Line(points={{-104,18},{-96,18}}, color={0,0,255}));
+  connect(r2n.n, c23.p) annotation (Line(points={{-80,18},{-40,18},{-40,24},{
+          -18,24},{-18,18}},
+                         color={0,0,255}));
+  connect(r3n.n, c23.n)
+    annotation (Line(points={{-80,-2},{-18,-2}}, color={0,0,255}));
+  connect(c10.n, c20.n) annotation (Line(points={{-66,22},{-66,-36},{-54,-36},
+          {-54,-32}}, color={0,0,255}));
+  connect(ground1.p, v3a.p) annotation (Line(points={{-18,-36},{-132,-36},{
+          -132,-2},{-124,-2}}, color={0,0,255}));
+  connect(ground.p, v3.p) annotation (Line(points={{86,-32},{20,-32},{20,-12},
+          {28,-12}}, color={0,0,255}));
+  connect(c20.p, c23.p) annotation (Line(points={{-54,-12},{-54,18},{-40,18},
+          {-40,24},{-18,24},{-18,18}}, color={0,0,255}));
+  connect(c30.p, c23.n)
+    annotation (Line(points={{-36,-12},{-36,-2},{-18,-2}}, color={0,0,255}));
+  annotation (
+    Diagram(coordinateSystem(extent={{-140,-60},{120,60}})),
+    Documentation(info="<html>
+<p>MSL uses, inside M_OLine, a network of physical capacitors to model capacitive effects across conductors.</p>
+<p>Insstead, in typical PowerSystems approach, a capacitance matrix is used, corresponding to the vector equation:</p>
+<p><b>i</b>=<i><b>C&nbsp;</b></i>d<b>v</b>/dt&nbsp;</p>
```

Comment:
Please omit usage of deprecated <b> tag, use <strong> instead. Similar, use <em> instead of <i>.
Optionally, <var> tag can be used for variables whereas <code> is typically used for an inline code snippet or e.g. for code parameters.